homeassistant/components/compit/strings.json

```
@@ -7,18 +7,29 @@
         "data": {
           "email": "[%key:common::config_flow::data::email%]",
           "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "email": "Your inext.compit.pl account email",
+          "password": "Password to your inext.compit.pl account"
```

```suggestion
          "email": "The email address of your inext.compit.pl account",
          "password": "The password of your inext.compit.pl account"
```
To make it consistent with how we currently define these.